packages/cashier/src/pages/withdrawal/withdrawal-crypto-receipt/withdrawal-crypto-receipt.tsx

```diff
@@ -1,41 +1,68 @@
 import React from 'react';
 import { Button, Clipboard, Icon, Text } from '@deriv/components';
-import { isMobile } from '@deriv/shared';
+import { useCryptoTransactions } from '@deriv/hooks';
+import { TModifiedTransaction } from '@deriv/hooks/src/useCryptoTransactions';
 import { observer, useStore } from '@deriv/stores';
 import { Localize, localize } from '@deriv/translations';
-import { useCashierStore } from '../../../stores/useCashierStores';
-import { TAccount } from '../../../types';
-import { getAccountText } from '../../../utils/utility';
+import { useCashierStore } from 'Stores/useCashierStores';
+import { TAccount } from 'Types';
+import { getAccountText } from 'Utils/utility';
+import { cryptoTransactionMapper } from '../../../modules/transactions-crypto/helpers';
 import './withdrawal-crypto-receipt.scss';
 
 type TWalletInformationProps = {
     account: TAccount;
     blockchain_address: string;
+    is_mobile: boolean;
 };
 
-const Status = () => {
+type TStatusProps = {
+    last_transaction:
+        | (TModifiedTransaction & {
+              is_deposit: boolean;
+              is_withdrawal: boolean;
+          })
+        | undefined;
+};
+
+const Status = ({ last_transaction }: TStatusProps) => {
+    const [status, setStatus] = React.useState({ status_color: 'warning', status_name: 'In review' });
+
+    const colorMap: { [key: string]: string } = {
+        warning: 'orange',
+        unsuccessful: 'red',
+        default: 'green',
+    };
+
+    React.useEffect(() => {
+        if (last_transaction) {
+            const { status_color, status_name } = cryptoTransactionMapper(last_transaction);
+            setStatus({ status_color, status_name });
+        }
+    }, [last_transaction]);
+
     return (
         <Text as='p' color='prominent' weight='bold' align='center' className='withdrawal-crypto-receipt__status'>
             <Icon
                 icon='IcCircleDynamicColor'
-                color='orange'
+                color={colorMap[status.status_color] || colorMap.default}
                 size={8}
                 className='withdrawal-crypto-receipt__status-icon'
             />
-            <Localize i18n_default_text='In review' />
+            {status.status_name}
```

Comment:
I think we need to add Localize to this `status.status_name`

Reply:
for this we dont need to, cause inside cryptoTransactionMapper, the status_name strings are already wrapped in localize